RELEASE.md

```diff
@@ -54,3 +54,20 @@ git push upstream v1.2.0
 
 ### Post-Steps
 Once the tag has been pushed the release action should run automatically. You can view the progress [here](https://github.com/operator-framework/operator-controller/actions/workflows/release.yaml). When finished, the release should then be available on the [releases page](https://github.com/operator-framework/operator-controller/releases).
+
+
+## Backporting Policy
+
+Mainly critical issue fixes are backported to the most recent minor release.
+Special backport requests can be discussed during the weekly Community meeting or via Slack channel;
+this does not guarantee an exceptional backport will be created.
+
+Occasionally non-critical issue fixes will be backported, either at an approver’s discretion or by request as noted above.
```

Comment:
IMO we do not have capacity for non-critical issue fix backport at this point of time. In future if the community grows to a large pool of contributors, may be we can start doing it then.

Reply:
This line allows us, at our discretion, to issue any patch release if needed. It provides the freedom and flexibility to do so, even for cases that may not be critical, medium, or low risk, but where we determine that a patch is necessary for any reason.

**We must retain the ability to release patches when we consider that is necessary no matter what.**

Reply:
> We must retain the ability to release patches whenever we consider it necessary
Sure

> This is also the policy for the SDK and other projects under the same umbrella

In that case , we just follow other projects under OLM as you mentioned.

Reply:
Yep, just following the other projects